### PR TITLE
Demo changes

### PR DESCRIPTION
**Pull request by jaspreet21anand**

### CODE REVIEW

File: app/views/admin/users/index.html.erb [hunk 1]
```diff
@@ -16,29 +17,20 @@
...
-  <th>Contact number</th>
-  <th>About me</th>
   <th>#Requests</th>
   <th>#Donations</th>
   <th>Enable/Disable</th>
-  <th colspan="2">Edit</th>
+  <th>Edit</th>
   <% @users.each do |user| %>
     <tr>
       <td><%= link_to user.name, admins_user_path(user) %></td>
```
Comment (ShubhamGupta) [inline app/views/admin/users/index.html.erb]: Add a function `display_name` in user class.

File: app/views/comments/_comment.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <% comment ||= @comment %>
-<li>
-  <strong><%= comment.user.name.capitalize %></strong> commented on <%= comment.created_at.strftime("%d %b %Y at %I:%M %p") %>
+<li class='highlight-grey-border round-corners'>
+  <strong>&nbsp;<%= comment.user.name.capitalize %></strong> commented on <%= comment.created_at.strftime("%d %b %Y at %I:%M %p") %>
```
Comment (ShubhamGupta) [inline app/views/comments/_comment.html.erb]: Set date formats app wide.

File: app/views/layouts/admin.html.erb [hunk 3]
```diff
@@ -7,6 +7,7 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
+  <% @admin = true %>
```
Comment (ShubhamGupta) [inline app/views/layouts/admin.html.erb]: Fix issues discussed in layouts.

**Comment by jaspreet21anand** [comment]
Please review again.

**Comment by jaspreet21anand** [comment]
Will fix JS in other branches

**Pull request merged**